ntbs-service/Models/ManualTestResult.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using ExpressiveAnnotations.Attributes;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.Validations;
+
+namespace ntbs_service.Models
+{
+    public class ManualTestResult
+    {
+        // Even for values which are non-nullable in db, we make them a nullable runtime type so 
+        // the Required attribute can be applied properly, producing correct error messages
```

Comment:
So what happens if you don't make them nullable - does it just give you a sql error rather than the error message? This will apply to my case too

Reply:
yes, i suppose it would. we never get there, because we validate the model before saving it to the db